map/display-map/gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem
```

Comment:
Same here, not sure whether the update here was intentional (gradle updates these automatically when you run the wrapper task, I think)